test/PreAMMBatcher-e2e.ts

```diff
@@ -62,7 +59,7 @@ describe("PreAMMBatcher-e2e", () => {
     walletTrader4,
     walletTrader5,
     walletTrader6,
-  ] = new MockProvider().getWallets();
+  ] = waffle.provider.getWallets();
```

Comment:
If I understand it right, `MockProvider()` is an interface with a fresh instance of `ganache-cli` (but I can't find any documentation on this, that's my guess). So this change implicitly requires us to use only a single global instance of `waffle`, right?

Reply:
The `waffle.provider` is a `MockProvider` that wraps the `buidlerevm` instead of `ganache`. We can always use `new MockProvider`, but then we don't get the `buidlerevm` goodies. (The tests before were using them for example).